### ISSUE TITLE
PKI firefox SEC_ERROR_INVALID_TIME tls error

### ISSUE THREAD

**Issue by bmpickford** [Nekroze/dab#87]
# Bug Report

## Describe the bug

Getting the error 'SEC_ERROR_INVALID_TIME' when trying to switch to dab. This is because dab uses a shorter TTL for each certificate than previously used, requiring the user to manually accept the certificate

## To Reproduce

If using a different service, switch to DAB and send a request to the api or view a web page

## Expected behavior

Certificates should already be accepted

## `dab version` output

```
benjaminm@benjaminm-pc:~$ dab version
Dab wrapper script appears to have an update available!
Dab Version: 27b4e5083265950bcdad9cc5840793456bf5056a
WARNING: No swap limit support
Server Version: 18.01.0-ce
Runtimes: runc
Kernel Version: 4.4.0-98-generic
Operating System: Ubuntu 16.04.3 LTS
Architecture: x86_64

```

**Comment by Nekroze** [comment]
according to https://wiki.mozilla.org/SecurityEngineering/x509Certs SEC_ERROR_INVALID_TIME means `A time field in a certificate has an invalid value` for which they suggest `Re-generate the certificate with valid encodings for time fields`.

So the issue is not related to TTL values but some kind of encoding of the date itself which is handled by vault. I am investigating if there is a way to change the default encoding vault uses to something firefox likes more.

**Comment by bmpickford** [comment]
Can confirm that it seems to be firefox specific, or at least works fine on Chrome

### Firefox version:
```
benjaminm@benjaminm-pc:~$ firefox -v
Mozilla Firefox 62.0.3
```

**Comment by Nekroze** [comment]
@bmpickford thanks for that, its extremely helpful.

**Comment by Nekroze** [comment]
I have tried mounting /etc/localtime into the vault to no effect.

**Comment by Nekroze** [comment]
This seems to not happen with the `0.10.x` versions of vault yet it still refuses to fully trust it without needing user input to accept the cert.

**Comment by Nekroze** [comment]
this does not appear to be an issues for vault `0.11.4` released 2 days ago, can you use #122 then run the following and try again:

```bash
$ dab services update
$ dab tools update
```

**Comment by bmpickford** [comment]
Can confirm the issue is resolved on my machine with https://github.com/Nekroze/dab/pull/122
